app/views/shared/email/_myCollection_block.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if Rails.env.production? %>
+    <p>Simply <%= link_to 'sign up', "https://www.artsy.net/signup?submissionId=#{@submission.id}", target: :_blank %> or <%= link_to 'log in', "https://www.artsy.net/login?submissionId=#{@submission.id}", target: :_blank %> to Artsy with the same email you used for this submission and visit our app.</p>
```

Comment:
We could also use a helper for this to return just the URL, to avoid duplication.
`<p>Simply <%= link_to 'sign up', "#{SomeHelper.environment_url("signup")}#{@submission.id}", target: :_blank %> or <%= link_to 'log in', "#{SomeHelper.environment_url("login")}#{@submission.id}", target: :_blank %> to Artsy with the same email you used for this submission and visit our app.</p>`